src/api/core/DatabaseServerlessHandler.ts

```diff
@@ -0,0 +1,21 @@
+import { Connection } from "typeorm";
+import Database from "../core/Database";
+
+
+export abstract class DatabaseServerlessHandler<T> {
+    
+    public connection: Connection;
+
+    abstract initializeDependencies?(event?: T): void
+
+    protected async initializeDatabase(): Promise<void> {
+        const database = new Database();
+        try {
```

Comment:
@bigLucas 
Is enough just use `try` to check if the database was successfully initialized?

Reply:
Yes, it is.